src/test/java/io/javalin/metrics/JavalinMetricsTest.kt

```diff
@@ -0,0 +1,21 @@
+package io.javalin.metrics
+
+import io.javalin.Javalin
+import io.javalin.util.TestUtil
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class JavalinMetricsTest {
```

Comment:
It doesn't look like these tests do anything now.

Reply:
deleted